src/OrchardCore/OrchardCore.OpenId.Core/Services/Managers/OpenIdApplicationManager.cs

```diff
@@ -17,16 +18,21 @@ namespace OrchardCore.OpenId.Services.Managers
     public class OpenIdApplicationManager<TApplication> : OpenIddictApplicationManager<TApplication>,
         IOpenIdApplicationManager where TApplication : class
     {
+        private readonly IStringLocalizer<IOpenIdApplicationManager> _localizer;
```

Comment:
We use `S` for a StringLocalizer.

Reply:
We agreed to use `S` in all the places as @agriffard mentioned before